Dockerfile

```diff
@@ -1,4 +1,4 @@
-# TODO: make this default to govuk-ruby once it's being pushed somewhere public
+FROM ruby:2.7.5
 # (unless we decide to use Bitnami instead)
 ARG base_image=ruby:2.7.2-slim-buster
```

Comment:
And this needs to be updated to `ARG base_image=ruby:2.7.5-slim-buster`

These PRs were automated. It works when the 1st line of the file is `FROM ruby:2.7.2` as it replaces it with `FROM ruby:2.7.5`, but when a variable is set and reused throughout the dockerfile it breaks it.